pkg/uroot/uroot.go

```diff
@@ -327,49 +344,59 @@ func (o *Opts) addSymlinkTo(logger ulog.Logger, archive *initramfs.Opts, command
 	return nil
 }
 
-// resolvePackagePath finds import paths for a single import path or directory string
-func resolvePackagePath(logger ulog.Logger, env golang.Environ, pkg string) ([]string, error) {
-	// Search the current working directory, as well GOROOT and GOPATHs
-	prefixes := append([]string{""}, env.SrcDirs()...)
+// resolvePackagePath finds import paths for a single import path or directory string.
+//
+// Possible options are:
+//
+//   ./foobar
+//   ./foobar/glob*
+//   UROOT_SOURCE/cmds/core/ip
```

Comment:
this doesn't make sense for the reader. "Possible options are" should be listing potential inputs for the function.

I think it should read

```
./foobar
./foobar/g*lob
cmds/core/ip with UROOT_SOURCE=/directory/to/u-root
cmds/core/g*lob with UROOT_SOURCE=/directory/to/u-root
```

because the input to `input` is `cmds/core/ip`, not `UROOT_SOURCE/cmds/core/ip`